package.json

```diff
@@ -2,7 +2,6 @@
   "name": "time-to-leave-website",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
   "keywords": [],
   "author": "",
   "license": "ISC",
```

Comment:
I'm curious why we chose this one here and gpl on the app

Reply:
Dunno, but I hadn't even noticed there was this license tag here until you pointed out. I'll update it to match the main repo